src/components/NowPlaying.tsx

```diff
@@ -2,28 +2,39 @@ import styles from "../css/app.module.scss";
 import React from "react";
 import getAudioFeatures from "../services/nowPlayingService";
 import { AudioFeaturesResponse } from "../types/spotify-web-api";
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
```

Comment:
I think the build is failing due to these imports. Have they been added properly to the `package.json` as a dependency?